### ISSUE TITLE
Sheet no longer shows multiple pages on single display if there is room

### ISSUE THREAD

**Issue by knownexus** [richardwilkes/gcs#548]
On previous Versions, if there was room on the screen, the sheet would show multiple pages
now it does not
![image](https://user-images.githubusercontent.com/19297623/192527344-fdde3c80-3785-44dc-9389-c1abeaefa8cb.png)
![image](https://user-images.githubusercontent.com/19297623/192527472-d89b3acc-0d40-47f5-8c9e-3903ecbe427a.png)

**Comment by richardwilkes** [comment]
Yes, that was a conscious design choice in the re-write. The effort required to break the page up in sane manner and yet allow all of the standard editing operations to function correctly over the split space was significant and slowed things down considerably. The new form just treats it more like a web page for display (i.e a single continuous page that is as tall as required), but does break it up when creating pdf or printing.

Given the reason it was removed, I'm unlikely to reverse the decision -- but as always, I'm willing to hear reasons why I should change my mind.

**Comment by knownexus** [comment]
I'm not certain how to change your mind on the matter
I just feel it should be an option available if someone is willing to have the app slow down for the sake of readability 
It's a feature i used constantly, as i have an ultra-wide monitor
and now (as show in the image above) it's annoying to read the sheet
That wasn't me stretching the app, thats just how it appears on my screen

It might be easier to logically split it up by section? such as splitting skills or spells into their own page,

It's just enough of a inconvenience for me to not want to upgrade
even if i do like the new look

**Comment by DirectXMan12** [comment]
ditto here, I've got multiple monitors, and tend to just shove my character sheet on a side monitor while playing, and it was nice to not have to scroll around a bunch.  IDK if this would make it easier (haven't gotten to look at the code much yet), but, at least for me, less complicated "splitting" where entire sections just shift over would likely still be helpful (e.g. the whole "trait/skill chunk" goes to a page, or the whole "notes" section goes to a page).  Alternatively, while likely more complicated, allowing users to manually split into another tab or something would also solve my usecase.

**Comment by fishface60** [comment]
It's one of those features I didn't know how useful it was until it enabled me to have large sheets and I no longer had it.

Making enough desktop space to have the whole sheet visible was very helpful for being able to memorise roughly where they'd be on the screen and being able to quickly refer to them.

Some form of reflowing to extend horizontally would be very helpful here even if it's not breaking the sections up like it was previously able to.

Being able to open up multiple split views of the same document like IDEs are capable of would also work, especially if the layout were saved between sessions.

I'm finding it particularly awkward at the moment without it given search has not yet been reimplemented.
